ee/server/service/vpp.go

```diff
@@ -83,19 +83,24 @@ func (svc *Service) BatchAssociateVPPApps(ctx context.Context, teamName string,
 
 	payloadsWithPlatform := make([]fleet.VPPBatchPayloadWithPlatform, 0, len(payloads))
 	for _, payload := range payloads {
+		// Currently only macOS is supported for self-service. Don't
+		// import vpp apps as self-service for ios or ipados
 		payloadsWithPlatform = append(payloadsWithPlatform, []fleet.VPPBatchPayloadWithPlatform{{
-			AppStoreID: payload.AppStoreID,
-			Platform:   fleet.IOSPlatform,
+			AppStoreID:  payload.AppStoreID,
+			SelfService: false,
+			Platform:    fleet.IOSPlatform,
```

Comment:
All apps get created for all 3 platforms? (was already like that, just curious for my own understanding)

Reply:
Yes I was confused about this as well. Apparently the platforms that are actually supported are filtered in the front end when we retrieve the metadata from itunes